packages/commerce-sdk-react/src/types.ts

```diff
@@ -0,0 +1,24 @@
+interface CommonHookResponse {
+    error: Error | undefined
+    isLoading: boolean
+}
+
+// These are the common params for all query hooks
+// it allows user to override configs for specific query
+interface QueryParams {
+    siteId?: string
+    locale?: string
+    currency?: string
+    organizationId?: string
+    shortCode?: string
+}
+
+interface QueryResponse<data> extends CommonHookResponse {
```

Comment:
```suggestion
interface QueryResponse<T> extends CommonHookResponse {
```
Convention for type names is to use `UpperCamelCase` and also to use `T` until things get complicated. (And even then, convention is to use single letters like `K` for keys, `R` for return type, etc.)

Reply:
Thanks!! fixed